config/locales/client.en.yml

```diff
@@ -1559,6 +1559,18 @@ en:
         success_via_admin: "We've sent an email to that address. The user will need to follow the confirmation instructions in the email."
         success_staff: "We've sent an email to your current address. Please follow the confirmation instructions."
         back_to_preferences: "Back to preferences"
+        confirm_success: "Your email has been updated."
+        confirm: "Confirm"
+        authorizing_new:
+          description: "Please confirm you would like your email address changed to:"
+          description_add: "Please confirm you would like to add an alternate email address:"
+        authorizing_old:
+          title: "Confirming new email address"
```

Comment:
This doesn't feel right, `authorizing_old.title` as "Confirming new email address" can be confusing for translators.

Reply:
:+1: updated